site/cds_rdm/permissions.py

```diff
@@ -53,6 +53,8 @@ class CDSRDMRecordPermissionPolicy(RDMRecordPermissionPolicy):
         )
     ]
 
+    can_manage_clc_sync = [Librarian(), SystemProcess()]
```

Comment:
```suggestion
    can_manage_clc_sync = [Librarian(), Administration(), SystemProcess()]
```
we need to be able to do it too :)